cmd/root.go

```diff
@@ -30,36 +29,31 @@ func Execute() {
 // init configures flags (both persistent across child commands and local to root)
 func init() {
 	cobra.OnInitialize(initConfig)
-	rootCmd.PersistentFlags().StringVar(&cfgFile, "config", "", "config file (default is ./terra-status-manager.yaml)")
+	rootCmd.PersistentFlags().StringVar(&cfgFile, "configuration", "", "configuration file (default is ./revere.yaml)")
 	rootCmd.PersistentFlags().BoolP("verbose", "v", false, "enable more verbose console output")
 	err := viper.BindPFlags(rootCmd.Flags())
 	cobra.CheckErr(err)
 }
 
-// initConfig reads in config file (flag or default) and ENV variables if set
+// initConfig reads in configuration file (flag or default) and ENV variables if set
 func initConfig() {
 	if cfgFile != "" {
 		viper.SetConfigFile(cfgFile)
 	} else {
 		viper.AddConfigPath(".")
-		viper.AddConfigPath("/etc/terra-status-manager/")
-		viper.SetConfigName("terra-status-manager")
+		viper.AddConfigPath("/etc/revere/")
+		viper.SetConfigName("revere")
 	}
 
-	// example: statuspage.apiKey overridden by env var TSM_STATUSPAGE_APIKEY
-	viper.SetEnvPrefix("TSM")
-	viper.SetEnvKeyReplacer(strings.NewReplacer(".", "_"))
```

Comment:
Nope. Viper's "unmarshalling" only works for values present in the original file. So _Viper_ will be made aware of values passed this way, but we never ask Viper directly--we have Viper unmarshall to a struct and go from there. It works even if there is a stub for the key in the config file, but that seems unintuitive. I'm keeping viper around because it actually gives us some really slick features with intelligently finding config files cross-platform (and secretly having support for json, toml, java properties, etc that'll still unmarshall to yaml)